hyperliquid/exchange.py

```diff
@@ -492,6 +602,20 @@ def usd_transfer(self, amount: float, destination: str) -> Any:
         )
 
     def spot_transfer(self, amount: float, destination: str, token: str) -> Any:
+        """
+        Initiate a spot transfer of a specific token to a specified destination address.
+
+        This function creates and signs a spot transfer action for a given token, then sends it
+        for processing.
+
+        Args:
+            amount (float): The amount of the token to transfer.
+            destination (str): The destination address (EVM compatible) for the transfer.
+            token (str): The identifier of the token to be transferred (e.g., "BTC", "ETH").
```

Comment:
These example tokens are not spot tokens. Also the token identifier is a different format
```suggestion
            token (str): The identifier of the token to be transferred (e.g., "PURR:0xc1fb593aeffbeb02f85e0308e9956a90").
```